docs/developer-guide/syntax-reference.rst

```diff
@@ -1153,6 +1153,20 @@ Example:
       WHERE ROWTIME >= 1510923225000
         AND ROWTIME <= 1510923228000;
 
+When writing logical expressions using ``ROWTIME``, ISO-8601 formatted datestrings can also be used to represent dates.
+For example, the above query is equivalent to the following:
+
+.. code:: sql
+
+    SELECT * FROM pageviews
+          WHERE ROWTIME >= '2017-11-17 04:53:45'
+            AND ROWTIME <= '2017-11-17 04:53:48';
+
+If the datestring is inexact, the rest of the timestamp is assumed to be padded with 0's.
+For example, ``ROWTIME = '2019-07-30 11:00'`` is equivalent to ``ROWTIME = '2019-07-30 11:00:00.0000'``.
+
+Note that the timestamps are interperted in the UTC timezone.
```

Comment:
UTC isn't going to work for everyone. We should support timezone information in the datetime string, but default to UTC is none is provided.